src/utils/ts/FocusManager.tsx

```diff
@@ -19,6 +19,7 @@ type FocusManagerProps = {
 
 const FocusManager: React.FC<FocusManagerProps> = ({ children, refs = undefined, defaultFocusedIndex }) => {
   const [focusedIndex, setFocusedIndex] = useState<number>(defaultFocusedIndex ?? 0);
+  const prevFocusedIndex = useRef<number>(focusedIndex);
```

Comment:
Maybe we call this variable `focusedIndexRef` because the name `prevFocusedIndex` is already used by the `focusPrevious`and `focusNext`. Another option would be `focusedIndexPreviousValue`, `focusedIndexOld`, `focusedIndexAux`

Reply:
The idea was to mimic `prevState` that we had in lifecycle hooks, but I see how this could be confused with `focusPrevious` function. I can rename it.

Reply:
It's just a bit confusing because when you mount the component there is not previous focused. I'm ok if the name stays as it is. I guess it's not a big deal.